src/components/pages/enterprise/training/training.jsx

```diff
@@ -34,6 +36,15 @@ const items = [
     buttonLink: 'https://isovalent.com/labs/',
     buttonTarget: '_blank',
   },
+  {
+    logoName: 'soloio',
+    title: 'Introduction to Cilium with Certification Option',
```

Comment:
```suggestion
    title: 'Introduction to Cilium',
```

We shouldn't imply any official certification (until we come up with something like the Cilium equivalent of the CKA of course!)